docs/use-cases/UC_CA_GUI.md

```diff
@@ -0,0 +1,47 @@
+Use Case: Running Chronoamperometry for a Two or Three Electrode Cell on the JUAMI Potentiostat through the graphical user interface. Intended User: Undergraduate (UG) Researcher at a University in Africa with Introductory Knowledge of Electrochemistry and Chronoamperometry, and limited experience with Python. The instructions in this UC assume that the student already has a two or three electrode cell set up and ready for testing (working electrode, counter electrode, and (optional) reference electrode are all in proper locations and connected to the potentiostat clips). 
+
+Program Setup:
+
+UG double clicks on the pytentiostat icon on the desktop
```

Comment:
This is better as a numbered list.  To do that put 1. at the beginning of each item and remove the empty lines.  So it would look like:
```
### Program Setup ###
1. UG double clicks on the pytentiostat icon on the desktop
1. Program displays a GUI
1. UG left clicks.....
```
And so on .

Reply:
Updated the file to be a numbered list. Looks much better now!